### PR TITLE
Landing and connexion page

### PR DESCRIPTION
**Pull request by NinaSwoop**
style 
index.html.twig change image background, add connexion button, add base line and logo with good cliping add some correction in the layout.html.twig for the nav

### CODE REVIEW

File: src/View/Home/index.html.twig [hunk 1]
```diff
@@ -1,10 +1,24 @@
...
+					<p class="logotxt">goldfish</p>
+					<img src="/assets/images/goldfish.png" class="logoimg" alt="logo goldfish">
+				</div>
+			</div>
+		</div>
+		<div class="row fixed-bottom mb-5">
+			<div class="form-signin">
+				<h3 class="baseline">Grâce à lui, vous ne passerez plus à côté des concerts de vos artistes favoris</h3>
+				<div class="button-connexion d-flex justify-content-center">
+					<button type="submit" class="connexionbutton btn btn-danger btn-md mb-5 d-flex justify-content-center">Connexion</button>
```
Comment (Dldvlpr) [inline src/View/Home/index.html.twig]: "connexionbutton" 

no corresponding class in css.

**Review by Dldvlpr — CHANGES_REQUESTED** [review]

File: src/View/Home/index.html.twig (hunk 1 above)
Comment (NinaSwoop) [reply src/View/Home/index.html.twig]: remove connexionbutton on class ok

File: database.sql [hunk 2]
```diff
@@ -16,7 +16,7 @@ create table artist (
      id int not null primary key auto_increment,
      name varchar(100) not null,
      style varchar(50) not null,
-     image varchar(255) not null
+     image varchar(255) not null,
```
Comment (aldaitzwild) [inline database.sql]: No this last coma, fails the query

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

**Review by aldaitzwild — CHANGES_REQUESTED** [review]
Conflict with dev

File: src/View/Home/index.html.twig (hunk 1 above)
Comment (aldaitzwild) [reply src/View/Home/index.html.twig]: No not ok

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

**Review by aldaitzwild — APPROVED** [review]

**Pull request merged**